icons/android/back-24.svg

```diff
@@ -1,3 +1,3 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. --><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M21 11H5.914l5.793-5.793a1 1 0 1 0-1.414-1.414l-7.5 7.5a1 1 0 0 0 0 1.414l7.5 7.5a1 1 0 0 0 1.414-1.414L5.914 13H21a1 1 0 0 0 0-2z"/></svg>
\ No newline at end of file
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. --><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.707 3.293a1 1 0 0 1 0 1.414L5.414 11H21a1 1 0 1 1 0 2H5.414l6.293 6.293a1 1 0 0 1-1.414 1.414l-8-8a1 1 0 0 1 0-1.414l8-8a1 1 0 0 1 1.414 0z" fill="#000"/></svg>
```

Comment:
Last I checked, `fill` should be set to `context-fill` and not `none` or `#000000`, or not set at all.

Reply:
@ExE-Boss iirc there is no `context-fill` on Android nor iOS, it's a Firefox for Desktop only feature.

The fill set to `none` is not necessary and it should be removed, thanks!

Reply:
`context-fill` is an [official SVG 2.0 feature](https://svgwg.org/svg2-draft/painting.html#TermContextElement), Firefox just has the [`-moz-context-properties` property](https://developer.mozilla.org/docs/Web/CSS/-moz-context-properties) to allow setting it on SVG images which are embedded in an HTML `<img>` tag (see also https://github.com/w3c/svgwg/issues/348).

Reply:
Yeah, thanks for the clarification ☺️  I meant that we don't use `context-fill` on mobile. Let me talk more with mobile engineers in order to understand how they can set the color of an icon if they import all the icons with just one package.